sdk/storage/azure-storage-blobs/test/blob_container_client_test.cpp

```diff
@@ -203,7 +203,7 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_TRUE(std::includes(listBlobs.begin(), listBlobs.end(), p1Blobs.begin(), p1Blobs.end()));
   }
 
-  TEST_F(BlobContainerClientTest, DISABLED_ListBlobsHierarchy)
```

Comment:
Is the service bug resolved?

Reply:
what was the bug?

Reply:
I dont know why it was disabled in the first place.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/860

Reply:
ut passed, so i think the bug is already fixed

Reply:
let's wait for the live test results. Maybe the region/tenant that you are targetting your UT to has this issue resolved.